MAINTAINERS

```diff
@@ -12,6 +12,8 @@
 "Jeffry Molanus",@gila,Independent
 "Kiran Mova",@kmova,VMware
 "Pawan Prakash Sharma",@pawanpraka1,Independent
+"Tiago Castro",@tiagolobocastro,DataCore Software
```

Comment:
@tiagolobocastro can you approve as well please.

Reply:
Yes sorry guess it makes sense for me to approve this too @kmova :)